app/src/app_config.py

```diff
@@ -30,7 +30,7 @@ def db_session(self) -> db.Session:
     # so they are not unique across different Phoenix instances.
     PROMPT_VERSIONS: dict = {
         "extract_supports": "UHJvbXB0VmVyc2lvbjoz",
-        "generate_referrals": "UHJvbXB0VmVyc2lvbjoxOQ==",
+        "generate_referrals": "UHJvbXB0VmVyc2lvbjoyOA==",
```

Comment:
[Updated prompt](http://localhost:6006/prompts/UHJvbXB0OjY=/playground) to return one JSON object and if-then logic to handle retry loop